feature-libs/organization/unit-order/styles/_unit-order.scss

```
@@ -24,6 +24,15 @@
     td {
       width: 16.6%;
       padding: 1rem 0;
+      max-width: 1px;
```

setting `max-width: 1px;` is messing up the mobile view. Since we need this only on table and mobile view we should wrap it in media-breakpoint
```suggestion
@include media-breakpoint-up(md) {
  max-width: 1px;
}
```

---

Thanks that's correct, I adapted this part.